docs/designers-developers/developers/tutorials/create-block/submitting-to-block-directory.md

```diff
@@ -0,0 +1,102 @@
+# Share your Block with the World
+
+So you've created an awesome block? Care to share?
+
+**Contents**:
+1. Help users understand your block
+2. Analyze your plugin
+3. Zip & Submit
+
+## Step 1: Help users understand your block
+
+Providing straightforward, easy to understand block information is important to the block directory and our end users.
+
+**Guidelines**:
+
+- Name your block based on what it does
+- Clearly describe your block
+- Add Keywords for all contexts
+- Choose the right category
+
+### Name your block based on what it does
+
+Users typically search the block directory within the Block Editor and do so in the context of a task. For example, when building their post, a user may search the Block Directory for an “image gallery”. Naming your block accordingly will help the Block Directory surface it when it's needed.
+
+**Not So Good**: WebTeam5 Image Works
+**Good**: Responsive Image Slider by WebTeam5
+
+**Question: What happens when there are multiple blocks with similar names?**
+Try your best to make your block's name functional and unique to make it stand out. Look for applicable synonyms or include a prefix if necessary.
+
+### Clearly describe your block
+
+The description really helps to communicate what your block does.The quicker a user understands how your block will help them, the more likely it is a user will use your block. Users will be reading your block's description within the Block Editor where space can be limited. Try to keep it short and concise.
+
+**Not So Good**: The best way to show images on your website using jQuery and CSS.
+**Good**: An easy to use, responsive, Image gallery block. 
+
```

Comment:
A responsive image gallery block. (easy to use well that is up to the user to decide.)